Jenkinsfile.promote

```diff
@@ -435,10 +435,9 @@ void mergeAndPush(String prLink) {
     mergeAndPush(prLink, getBuildBranch())
 }
 
-void tagLatest() {
+void tagLatest(String repo) {
```

Comment:
with latest change, I guess the parameter is not more needed.

Reply:
Thanks you, I wish I had the same catchy eyes as you!

Reply:
always harder to see when it is your own code ;)